### PR TITLE
TASK: Remove tooltip in the document and context tree

### PR DESCRIPTION
**Pull request by dfeyer**

### CODE REVIEW

**Comment by kdambekalns** [comment]
[PR: TASK: Remove tooltip in the document and context tree](https://trello.com/c/Hr624MMI/2-pr-task-remove-tooltip-in-the-document-and-context-tree)

**Comment by kdambekalns** [comment]
[PR: BUGFIX: Hide node tree tooltips on page scroll](https://trello.com/c/VnfT3hag/1-pr-bugfix-hide-node-tree-tooltips-on-page-scroll)

**Comment by dfeyer** [comment]
@aertmann @mficzel The native title work fine now

++

File: Neos.Neos/Resources/Public/JavaScript/Content/Navigate/AbstractNodeTree.js [hunk 1]
```diff
@@ -673,7 +671,6 @@ define(
 						node.data.key = result.data.key;
 						node.data.name = result.data.name;
 						node.data.title = result.data.title;
-						node.data.tooltip = result.data.tooltip;
```
Comment (aertmann) [inline Neos.Neos/Resources/Public/JavaScript/Content/Navigate/AbstractNodeTree.js]: this should still be there..

File: Neos.Neos/Resources/Public/JavaScript/Content/Navigate/NodeTree.js [hunk 2]
```diff
@@ -164,7 +164,6 @@ define(
 					children: [
 						{
 							title: siteName,
-							tooltip: siteName + (nodeTypeConfiguration ? ' (' + I18n.translate(nodeTypeConfiguration.label) + ')' : ''),
```
Comment (aertmann) [inline Neos.Neos/Resources/Public/JavaScript/Content/Navigate/NodeTree.js]: this should still be there as well..

**Comment by dfeyer** [comment]
Fix last comments by @aertmann

**Comment by dfeyer** [comment]
@aertmann Any chance for a last review ?

**Pull request merged**